tests/test.js

```diff
@@ -16,45 +15,114 @@ describe("sqlparser", function () {
       );
       throw new Error("wrong error");
     } catch (err) {
-      assert.equal(err.message, "syntax error at position 40 near 'blah'");
+      assert.strictEqual(
+        err.message,
+        "syntax error at position 40 near 'blah'"
+      );
     }
   });
 
+  test("create type", async function () {
+    const { type } = await globalThis.sqlparser.normalize(
+      "create table blah_5_ (id int, image blob, description text);"
+    );
+    assert.strictEqual(type, "create");
+  });
+
+  test("read type", async function () {
+    const { type } = await globalThis.sqlparser.normalize(
+      "select * from fake_table_1 where something='nothing';"
+    );
+    assert.strictEqual(type, "read");
+  });
+
+  test("write type", async function () {
+    const { type } = await globalThis.sqlparser.normalize(
+      "insert into blah_5_ values (1, 'three', 'something');"
+    );
+    assert.strictEqual(type, "write");
+  });
+
   test("missing args", async function () {
     try {
+      // @ts-ignore error
       await globalThis.sqlparser.normalize();
       throw new Error("wrong error");
     } catch (err) {
-      assert.equal(err.message, "missing required argument 'statement'");
+      assert.strictEqual(err.message, "missing required argument 'statement'");
     }
   });
 
   test("passing", async function () {
-    const [result] = await globalThis.sqlparser.normalize(
+    const { type, statements } = await globalThis.sqlparser.normalize(
```

Comment:
this pattern is nice, because you can grab just the normalized statements, or the type, or both!